packages/schemas-rtdb/src/helpers.ts

```diff
@@ -0,0 +1,2 @@
+export const getSchemasRefPath = (ref: string, env: string) =>
+  `/flamelink/environments/${env}/schemas/${ref || ''}`
```

Comment:
can you set the default value for the ref in the method signature?

Reply:
We could, but would then have to set for `env` as well because it comes after `ref`. And `ref` would then only be an empty string if it is `undefined` and not `null` for instance.

Most of this would not happen though because this helper is only called from the `schemas.ref()` method which ensures these arguments are set.